estate/views/estate_menus.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_property_menu_action" name="Properties" action="estate_property_action"/>
+        <menuitem id="estate_settings_menu" name="Settings">
+            <menuitem id="estate_property_type_menu_action" action="estate_property_type_action"/>
+            <menuitem id="estate_property_tag_menu_action" action="estate_property_tag_action"/>
+        </menuitem>
+    </menuitem>
```

Comment:
for the exercise: make one line per menuitems, not using encapsulation
but using parent_id instead.
this will help you when adding menuitems on bridge modules or modules depending on the module where you want to put a new menu item